README.md

```diff
@@ -3,6 +3,12 @@
 
 The app exposes a REST API to control wifi connectivity. Using the right network interface, it is possible to connect and disconnect from wifi networks, but also store and manage network configurations in /etc/network/interfaces and in a Sqlite3 database.
 
+The app runs with [Python 2.7.9][1], but ideally should work with [all Python 2.7 releases][2].
```

Comment:
It's enough to say 2.7.
We'll double-check then that we also support python3, but don't worry now about it.